jpsreport/scripts/plot_Nt.py

```diff
@@ -0,0 +1,81 @@
+# run script without arguments for help
+import sys
+import os
+from xml.dom.minidom import parse
+import glob
+import argparse
+import numpy as np
+import matplotlib.pyplot as plt
+
+def getParserArgs():
+        parser = argparse.ArgumentParser(description='Plot N-t graph measured by method A')
+        parser.add_argument("-f", "--inifile", help='jpsreport inifile (required)', required=True)
+        parser.add_argument("-i", "--id", help='id of the measurement area (optional)', nargs='?', default=-1, type=int)
+        args = parser.parse_args()
+        return args
+
+def plot_nt(measurement_id):
+    """
+    plot N(t) curves (Method A)
+    args:
+    measurement area id (int)
+    """
+    legend = False
+    fig = plt.figure(figsize=(16, 16), dpi=100)
+    ax1 = fig.add_subplot(111,aspect='auto')
+    plt.rc("font", size=30)
```

Comment:
Do we need it?

Reply:
Let me clarify my trivial question for line 26 and 27: 
I do not consider the setting of the font size and the font type to be necessary for a plot for overview. The design should be left to the user. I do not consider the setting of the font size and the font type to be necessary for a plot for overview. The design should be left to the user. In my opinion the scripts should be kept general. If we specify design elements, we should point them out and explain them. The search for non-standard settings can be time-consuming.

The file was removed with commit a8496940. The question is therefore invalid. I will resolve this conservation.